includes/wc-product-functions.php

```diff
@@ -434,6 +434,9 @@ function wc_scheduled_sales() {
 
 			// Sync parent
 			if ( $parent ) {
+                                // Clear prices transient for variable products.
```

Comment:
I haven't tested or reviewed the issue yet, but from a quick look - can you fix the spacing on this comment so it aligns with `delete_transient`?